docs/working-with-projects/working-with-images/sign-images.md

```diff
@@ -63,37 +63,166 @@ Note that Harbor's [garbage collection](../../administration/garbage-collection/
 
-## Use Notation(experimental) to sign artifacts with distribution spec v1.1 mode
-In order to use Notation, generate a certificate for signing artifacts.
+## Use Notation to sign and verify artifacts with distribution spec v1.1 mode
 
+Harbor supports configuring `http` and `https` domain. For security consideration, we use `https` domain name for secure image signing and verification in this section.
+
+###  Configure domain name
+* Prepare a registered domain name. `myharbor-registry.online` will be used as the domain name for this illustration.
+
+* Point a DNS record to the IPV4 address of your Harbor server.
+
+### Installing Certbot
+To enable secure HTTPS access to our Harbor registry, we need to obtain an SSL certificate from a Certificate Authority. Certbot will be used to obtain an SSL Certificate from [Let’s Encrypt](https://letsencrypt.org/).
+
+Use `apt` or `snap` to install certbot.
+
+```shell
+sudo apt install --classic certbot
+sudo certbot certonly --standalone -d myharbor-registry.online
 ```
-notation cert generate-test --default "wabbit-networks.io"
+After Certbot successfully obtains a certificate for your domain, it will provide the file paths where the certificate and key files are stored.
+
+![certbot](../../../img/certbot.png)
+
+Please take note of these paths, as they will be required in a subsequent step.
+
+### Configure Harbor domain
+After downloading Harbor release package and extracting it. Update the `harbor.yml` file by setting the hostname to your domain name like in the example below:
+
+`hostname: myharbor-registry.online`
+
+Replace `myharbor-registry.online` with the domain name obtained from your domain registrar.
+
+Next, under `https`, update the path of cert and key files to where the certificate and key files are stored. For Example:
+
+![Harbor configuration](../../../img/config.png)
+
+### Run the Installation Script
+Execute the Harbor installation script:
+
+`sudo ./install.sh`
+
+When successful, you should see a similar message like below:
+
+![harbor installation script](../../../img/script.png)
+
+### Log in to Harbor
+Log in to Harbor using the following Docker command:
+
+`docker login -u admin -p Harbor12345 myharbor-registry.online`
+
+Replace `myharbor-registry.online` with your domain’s address and navigate to https://myharbor-registry.online in your browser to access the Harbor interface, then login using the default Harbor credentials.
+
+![Harbor registry dashboard](../../../img/dashboard.png)
+
+### Install Notation CLI
+Install the latest version on Linux. Follow the [installation guide](https://notaryproject.dev/docs/user-guides/installation/cli/) for other platforms and methods.
+
+`brew install notation`
+
+### Build and push an image to Harbor Registry
```

Comment:
This section can be simplified and you can assume there is an image pushed to Harbor already. Just use `<harbor-domain>:<image-reference>` as the signing object. Make sure this document focuses on how to sign an existing image in Harbor.